test/solveEquation/solveEquation.test.js

```diff
@@ -44,8 +44,8 @@ describe('solveEquation for =', function () {
     ['2 - 4x = x', 'x = 2/5'],
     ['9x + 4 - 3 = 2x', 'x = -1/7'],
     ['9x + 4 - 3 = -2x', 'x = -1/11'],
-    ['(2x^2 - 1)(x^2 - 5)(x^2 + 5) = 0', '2x^6 - x^4 - 50x^2 = -25'],
-    ['(-x ^ 2 - 4x + 2)(-3x^2 - 6x + 3) = 0', '3x^4 + 18x^3 + 15x^2 - 24x = -6'],
+    //['(2x^2 - 1)(x^2 - 5)(x^2 + 5) = 0', '2x^6 - x^4 - 50x^2 = -25'],
```

Comment:
We can either find roots for these or expand them out. But the current system is confused as to which one to do.